src/styles/components/_Articlebody.scss

```diff
@@ -0,0 +1,12 @@
+.social-icon {
+    width: 1.5rem;
+}
+.article-body {
+    line-height: 2rem;
+    font-weight: 300;
+}
+@media screen and (max-width: 575px) {
+    .article-title {
+        font-size: 6vw;
+    }
+}
```

Comment:
Great job. But please take care of the EOL.

Reply:
Ok thanks